Shared/sdk/SharedUtil.Crypto.h

```diff
@@ -28,4 +30,56 @@ namespace SharedUtil
 
         return result;
     }
+
+    inline SString Aes128encode(const SString& sData, const SString& sKey, const SString& sIv)
+    {
+		SString result;
+
+		CryptoPP::byte iv[ CryptoPP::AES::BLOCKSIZE ];
+		CryptoPP::byte key[ CryptoPP::AES::DEFAULT_KEYLENGTH ];
+
+		memcpy(iv, sIv.c_str(), CryptoPP::AES::BLOCKSIZE );
+		memcpy(key, sKey.c_str(), CryptoPP::AES::DEFAULT_KEYLENGTH);
```

Comment:
I dont think you have to use this array. In theory passing in the string + its length should be fine.

Reply:
Fixed